source/style-guide/writing/write-for-accessibility.txt

```diff
@@ -61,16 +61,19 @@ to” or “above” might not provide the correct context.
    * - Click **Save**.
      - Click the green button.
        Click the round button.
-   * - *Where an element appears more than once in the UI:*
-       1. Click **Clusters**.
-       2. Click **...** next to the cluster you want to modify.
-       *or*
-       *Where an element appears once in the UI:*
-       1. Click **Clusters**.
-       2. Click **...**.
-     - 1. Click **...**.
-       *or*
-       2. In the clusters view, click the **...** button next to the cluster you want to modify.
+   * - | *Where an element appears more than once in the UI:*
```

Comment:
**nit:** We use the icon for the ellipsis (:icon-mms:`ellipsis`), so this may need to be revisited across the style guide.

Reply:
[Noted](https://docs.google.com/document/d/1gwg44VP4_P-hDn-QhLEDOpon-ZnzfEFLxmIa4qy1kgQ/edit?usp=sharing).